.github/workflows/website.yml

```diff
@@ -1,9 +1,7 @@
-name: Deploy website to Pages
+name: Build and Deploy website to Pages
 
 on:
-  push: # Runs on pushes targeting the default branch
-    branches:
-      - main
+  push: # Runs on pushes targeting any branch
```

Comment:
we also need to check for pull requests — have a look at how it's done in `ci.yml`

Reply:
Thanks, should be fixed now

Reply:
I was just wondering if there are cases not covered by on `push` and covered by on `pull_request`, maybe when doing changes using GitHub editor ?

Reply:
Ah forgot about forks xD never mind.